sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -123,6 +123,9 @@ export interface SpanOptions {
     attributes?: {
         [key: string]: unknown;
     };
+    kind?: SpanKind;
+    // Warning: (ae-forgotten-export) The symbol "Link" needs to be exported by the entry point index.d.ts
```

Comment:
Oh no, missing export!

Reply:
Fixed _but_ I'm starting to think we should just get out of this "intermediate" interface business. Most of the code after this and a few other in-process PRs, should make it so most code won't care what the underlying data model is and just pass it along.

Reply:
(not for this PR, but maybe soon after)